<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>772</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>622</width>
    <height>446</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>622</width>
    <height>446</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Печать</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QListWidget" name="listWidget1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>251</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPrint">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>170</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>ПЕЧАТЬ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>20</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Введите дату начала и конца уволнения </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_start">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_end">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>120</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ПО</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>251</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Список кадет</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Список увольняемых</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="add_cadetprint_button">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>330</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить кадета
 в список увольняемых</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_cadetprint_button">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>410</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить кадета 
из списка увольняемых</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_cadet_button">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>490</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить кадета
из списка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_addcadet">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>610</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_cadet_button">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>600</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить кадета 
в список</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clean_BD_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>707</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить базу данных</string>
   </property>
  </widget>
  <zorder>listWidget1</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_start</zorder>
  <zorder>lineEdit_end</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>ButtonPrint</zorder>
  <zorder>listWidget2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>add_cadetprint_button</zorder>
  <zorder>del_cadetprint_button</zorder>
  <zorder>del_cadet_button</zorder>
  <zorder>lineEdit_addcadet</zorder>
  <zorder>add_cadet_button</zorder>
  <zorder>clean_BD_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
